### ISSUE TITLE
1.00 Installs manpage to /usr/man instead of /usr/share/man

### ISSUE THREAD

**Issue by ametzler** [PhilipHazel/SDoP#1]
Hello,

1.00 stopped letting autoconf locate the correct manpage install directory and installs to @prefix@/man. Please change back to using @mandir@.

TIA, cu Andreas

[20_fix_man_install_path.diff.txt](https://github.com/PhilipHazel/SDoP/files/11064963/20_fix_man_install_path.diff.txt)

**Comment by PhilipHazel** [comment]
My apologies. I don't know why I made that change; I clearly wasn't thinking right about autoconf. I have applied and committed the patch. Do think I should put out a 1.01 release with just this patch? I don't think there are many SDoP users. // Thanks for noticing the move to GitHub. I've been tidying up my world and it seemed sensible to get everything in one place (not to mention the usefulness of having all the sources managed by git).
